am-lib/src/main/java/uk/gov/hmcts/reform/amlib/DefaultRoleSetupImportServiceImpl.java

```diff
@@ -60,6 +61,16 @@ public DefaultRoleSetupImportServiceImpl(DataSource dataSource) {
             .installPlugin(new SqlObjectPlugin());
     }
 
+    /**
+     * This constructor is recommended when you want to use existing transaction and not to span new transaction.
+     *
+     * @param transactionAwareDataSourceProxy TransactionAwareDataSourceProxy
+     */
+    public DefaultRoleSetupImportServiceImpl(TransactionAwareDataSourceProxy transactionAwareDataSourceProxy) {
+        this.jdbi = Jdbi.create(transactionAwareDataSourceProxy)
```

Comment:
Please change comment as below - 
This constructor is recommended when you want to use existing transaction from the calling service such as CCD and do not want to span a new transaction with in AM scope